### PR TITLE
zh: backup-to-s3, deploy-on-aws-eks: various small changes

### PR DESCRIPTION
**Pull request by qiancai**
### What is changed, added, or deleted? (Required)

- Use `tidb-cluster` as namespace instead of `test1`.
  This matches the namespace created by
  https://docs.pingcap.com/tidb-in-kubernetes/stable/deploy-on-aws-eks#create-namespace
    
- Change link to raw version of `backup-rbac.yaml` and directly specify
    it in the `kubectl` command
    
- Try to make the database permissions easier to read and apply.
    
- Add a note about the different methods for authentication
    
- Add information about describing backup jobs
    
- Add info about running an ad-hoc backup job again by first deleting it.
    
- Add info about verifying AWS access
    
- Add info about name constraints for the cluster and node group names
  This can save a lot of time as it otherwise fails late in the process if `_` is used.
    
- Change the expected time for cluster creating from 5 to 10 minutes to 5 to 20 minutes based on my experience.

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v1.1 (TiDB Operator 1.1 versions)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-tidb-operator/pull/1210
- Other reference link(s): 

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: zh/backup-to-s3.md [hunk 1]
```diff
@@ -15,16 +15,16 @@ aliases: ['/docs-cn/tidb-in-kubernetes/dev/backup-to-s3/']
 
 Ad-hoc 全量备份通过创建一个自定义的 `Backup` custom resource (CR) 对象来描述一次备份。TiDB Operator 根据这个 `Backup` 对象来完成具体的备份过程。如果备份过程中出现错误，程序不会自动重试，此时需要手动处理。
 
-目前兼容 S3 的存储中，Ceph 和 Amazon S3 经测试可正常工作。下文对 Ceph 和 Amazon S3 这两种存储的使用进行描述。本文档提供如下备份示例。示例假设对部署在 Kubernetes `test1` 这个 namespace 中的 TiDB 集群 `demo1` 进行数据备份，下面是具体操作过程。
+目前兼容 S3 的存储中，Ceph 和 Amazon S3 经测试可正常工作。下文提供了如何将 TiDB 集群的数据备份到 Ceph 和 Amazon S3 这两种存储的示例。示例假设对部署在 Kubernetes `tidb-cluster` 这个 namespace 中的 TiDB 集群 `demo1` 进行数据备份，以下是具体的操作过程。
 
 ### Ad-hoc 全量备份环境准备
 
-1. 下载文件 [backup-rbac.yaml](https://github.com/pingcap/tidb-operator/blob/master/manifests/backup/backup-rbac.yaml)，并执行以下命令在 `test1` 这个 namespace 中创建备份需要的 RBAC 相关资源：
+1. 执行以下命令，根据 [backup-rbac.yaml](https://raw.githubusercontent.com/pingcap/tidb-operator/master/manifests/backup/backup-rbac.yaml) 在`tidb-cluster`命名空间创建基于角色的访问控制（RBAC）资源。
```
Comment (qiancai) [inline zh/backup-to-s3.md:22]:
```suggestion
1. 执行以下命令，根据 [backup-rbac.yaml](https://raw.githubusercontent.com/pingcap/tidb-operator/master/manifests/backup/backup-rbac.yaml) 在 `tidb-cluster` 命名空间创建基于角色的访问控制（RBAC）资源。
```

File: zh/backup-to-s3.md [hunk 2]
```diff
@@ -247,9 +267,19 @@ options:
...
+
+要了解一个 Backup job 的详细信息，请使用以下命令。对于此命令中的 `$backup_job_name`，请使用上一条命令输出中的名称。
+
+{{< copyable "shell-regular" >}}
+
+``shell
+kubectl describe bk -n tidb-cluster $backup_job_name
 ```
 
+如果要再次运行 ad-hoc 备份，你需要[删除备份的 Backup CR](backup-restore-overview.md#删除备份的-backup-cr)并重新创建。
```
Comment (qiancai) [inline zh/backup-to-s3.md:281]:
```suggestion
如果要再次运行 ad-hoc 备份，你需要[删除备份的 Backup CR](backup-restore-overview.md#删除备份的-backup-cr) 并重新创建。
```

File: zh/backup-to-s3.md [hunk 3]
```diff
@@ -247,9 +267,19 @@ options:
 {{< copyable "shell-regular" >}}
 
 ```shell
-kubectl get bk -n test1 -owide
+kubectl get bk -n tidb-cluster -owide
+```
+
+要了解一个 Backup job 的详细信息，请使用以下命令。对于此命令中的 `$backup_job_name`，请使用上一条命令输出中的名称。
```
Comment (qiancai) [inline zh/backup-to-s3.md:273]:
```suggestion
要获取一个 Backup job 的详细信息，请使用以下命令。对于此命令中的 `$backup_job_name`，请使用上一条命令输出中的名称。
```

File: zh/backup-to-s3.md [hunk 4]
```diff
@@ -247,9 +267,19 @@ options:
...
 ```shell
-kubectl get bk -n test1 -owide
+kubectl get bk -n tidb-cluster -owide
+```
+
+要获取一个 Backup job 的详细信息，请使用以下命令。对于此命令中的 `$backup_job_name`，请使用上一条命令输出中的名称。
+
+{{< copyable "shell-regular" >}}
+
+``shell
```
Comment (csuzhangxc) [inline zh/backup-to-s3.md:277]:
```suggestion
```shell
```

**Review by DanielZhangQD — APPROVED** [review]
LGTM

**Review by csuzhangxc — APPROVED** [review]
LGTM

File: zh/backup-to-s3.md [hunk 5]
```diff
@@ -15,16 +15,16 @@ aliases: ['/docs-cn/tidb-in-kubernetes/dev/backup-to-s3/']
 
 Ad-hoc 全量备份通过创建一个自定义的 `Backup` custom resource (CR) 对象来描述一次备份。TiDB Operator 根据这个 `Backup` 对象来完成具体的备份过程。如果备份过程中出现错误，程序不会自动重试，此时需要手动处理。
 
-目前兼容 S3 的存储中，Ceph 和 Amazon S3 经测试可正常工作。下文对 Ceph 和 Amazon S3 这两种存储的使用进行描述。本文档提供如下备份示例。示例假设对部署在 Kubernetes `test1` 这个 namespace 中的 TiDB 集群 `demo1` 进行数据备份，下面是具体操作过程。
+目前兼容 S3 的存储中，Ceph 和 Amazon S3 经测试可正常工作。下文提供了如何将 TiDB 集群的数据备份到 Ceph 和 Amazon S3 这两种存储的示例。示例假设对部署在 Kubernetes `tidb-cluster` 这个 namespace 中的 TiDB 集群 `demo1` 进行数据备份，以下是具体的操作过程。
 
 ### Ad-hoc 全量备份环境准备
 
-1. 下载文件 [backup-rbac.yaml](https://github.com/pingcap/tidb-operator/blob/master/manifests/backup/backup-rbac.yaml)，并执行以下命令在 `test1` 这个 namespace 中创建备份需要的 RBAC 相关资源：
+1. 执行以下命令，根据 [backup-rbac.yaml](https://raw.githubusercontent.com/pingcap/tidb-operator/master/manifests/backup/backup-rbac.yaml) 在 `tidb-cluster` 命名空间创建基于角色的访问控制（RBAC）资源。
```
Comment (TomShawn) [inline zh/backup-to-s3.md:22]:
```suggestion
1. 执行以下命令，根据 [backup-rbac.yaml](https://raw.githubusercontent.com/pingcap/tidb-operator/master/manifests/backup/backup-rbac.yaml) 在 `tidb-cluster` 命名空间创建基于角色的访问控制 (RBAC) 资源。
```

File: zh/backup-to-s3.md [hunk 6]
```diff
@@ -247,9 +267,19 @@ options:
...
 
+要获取一个 Backup job 的详细信息，请使用以下命令。对于此命令中的 `$backup_job_name`，请使用上一条命令输出中的名称。
+
+{{< copyable "shell-regular" >}}
+
+```shell
+kubectl describe bk -n tidb-cluster $backup_job_name
+```
+
+如果要再次运行 ad-hoc 备份，你需要[删除备份的 Backup CR](backup-restore-overview.md#删除备份的-backup-cr) 并重新创建。
```
Comment (TomShawn) [inline zh/backup-to-s3.md:281]: 文档中其他地方都是首字母大写
```suggestion
如果要再次运行 Ad-hoc 备份，你需要[删除备份的 Backup CR](backup-restore-overview.md#删除备份的-backup-cr) 并重新创建。
```

**Review by TomShawn — APPROVED** [review]
LGTM
Only minor comments

**Pull request merged**